projects/mainframe.md

```diff
@@ -0,0 +1,106 @@
+-   Capturing infrastructure and application telemetry from a mainframe incurs a minimal performance impact and allows for different deployment scenarios of the OpenTelemetry Collector to minimize both mainframe CPU costs and mainframe network egress costs.
+
+## Deliverables
+
+-   Definition of semantic conventions that capture the concepts of the mainframe including hardware, virtualization, z/OS and subsystems, Linux on IBM Z by aligning to and extending the existing semantic conventions across resources, metrics, logs, and traces. The enhancements of the semantic conventions are assumed to be generically applicable to the mainframe and apply to any OS on the mainframe incl. TPF and z/VSE.
+-   Alignment of semantic conventions for virtualization between mainframe and other virtualizations solutions, as far as feasible and reasonable
+-   Description of mapping between names and mainframe terms (for conventions not specifically defined as mainframe extensions)
+-   Definition of a curated, most meaningful set of metrics (out of the thousands of available metrics) to be fully supported by the semantic conventions.
+-   Description and definition of semantic conventions for metrics determined as result from aggregation across system topology to simplify handling in observability backends
+-   Definition of approach for supporting instrumentation of mainframe-specific programming languages
+-   Definition of approach for supporting instrumentation of mainframe-specific logs
+-   Definition of approach for supporting trace context propagation
+-   Code instrumentation (SDKs) for mainframe-specific programming languages and runtimes
+
+## Staffing / Help wanted
+
+This group will welcome contributors from various areas:
+-   We look for participation from mainframe experts of all kinds: observability and monitoring experts, performance experts, virtualization experts, architects, and users of observability solutions for the mainframe and those who want to extend, simplify, and improve their current observability solution to the mainframe.
+-   We welcome developers working with Assembler, COBOL, PL/1, and REXX, and engineers willing to contribute with the instrumentation for mainframe-specific programming languages.
+-   We invite everybody to join who is passionate about the mainframe.
+-   Two project leads who are willing to drive the project and address any issues which are not handled by other project members.
+-   At least two sponsoring TC (Technical Committee) members. TC sponsors are dedicated to attending meetings, reviewing proposals, and in general being aware of the state of the project and its technical details. TC sponsors guide the project through the spec process, keep the tracking issue up to date, and help to ensure that relevant community members provide input at the appropriate times.
+-   Contributors from Semantic Conventions WG: \<to be added\>
+-   Maintainers or approvers: \<to be added\>
```

Comment:
I think the new process requires that the staffing is figured out before the [project is started](https://github.com/open-telemetry/community/blob/main/project-template.md#required-staffing). From this doc:

>Projects cannot be started until the following participants have been identified

It would be useful to start with finding TC sponsors since I know most TC members are already sponsoring other projects and likely won't be available.

Reply:
That's correct @tigrannajaryan. We have plenty of engineers from IBM and other firms who are willing to work on this, but we'll need some level of TC engagement. @dyladan, @alolita, and myself are all from the GC and we're part of this group. I don't expect this project to need as much TC attention as others that require spec changes like the semantic SIGs or RUM (which required new semantics for new scenarios).

I'm reaching out to the TC now. We weren't certain if the issue was meant to be filed at the same time as contacting the TC, or after.

Reply:
Added GC sponsors, and statement with respect to TC support.